legal-api/src/legal_api/models/business.py

```diff
@@ -335,9 +336,8 @@ def json(self):
         if self.tax_id:
             d['taxId'] = self.tax_id
         if self.state_filing_id:
-            base_url = current_app.config.get('LEGAL_API_BASE_URL')
             d['stateFiling'] = f'{base_url}/{self.identifier}/filings/{self.state_filing_id}'
-
+        d['summary'] = f'{base_url}/{self.identifier}/summary'
```

Comment:
should the summary be in the documents listing?
would it be an alternate view of the /businesses/identifier?output=summary as the default output would be json?

Reply:
It need not be. The summary had  more information like all the alterations that happened previously. We dont get from any of the existing endpoints. All other info can be retrieved from existing endpoints.

I will move the code in the new endpoint to the existing businesses endpoint and delete the new endpoint + remove thhe url from the business json